actions-runner/Dockerfile

```diff
@@ -20,5 +23,10 @@ COPY --from=build /usr/bin/curl /usr/bin/curl
 COPY --from=build /usr/bin/unzip /usr/bin/unzip
 COPY --from=build /usr/bin/zip /usr/bin/zip
 COPY --from=build /usr/bin/ssh* /usr/bin/
-COPY --from=build /usr/bin/aws /usr/bin/aws
+COPY --from=build /usr/bin/kubectl /usr/bin/kubectl
 COPY --from=build /usr/lib/ /usr/lib/
+COPY --from=aws-cli /usr/local/aws-cli/ /usr/local/aws-cli/
+COPY --from=aws-cli /lib64/ld-linux-* /lib64/
```

Comment:
Hmm, are you sure this won't conflict with other things maybe? 🤔 

Can we instead install ldlinux as package? Now you're copying libs from Amazon Linux base to an Ubuntu one

Reply:
While its indeed copying a lib between distros i prefer to do it this way then to do it in a more hacky way (eg https://github.com/skyscrapers/terragrunt-resource/blob/db15f03f745396e0a66d87217db1f9edbd55c9ef/docker-prod/Dockerfile#L9-L27)

Reply:
The part you've linked won't be needed anyway, that's due to alpine not coming with glibc but musl, which is not applicable here.

Instead I think you should just run the installer instead of copying from the upstream image, eg.

- https://github.com/skyscrapers/terragrunt-resource/blob/master/docker-prod/Dockerfile#L28-L42
- https://github.com/skyscrapers/server-images/blob/master/concourse-worker/scripts/setup.sh#L27-L30

I mostly want to avoid later problems by mixing libraries.

Reply:
Handled via Slack, no aws needed after all.